test/test_middleware.rb

```diff
@@ -23,7 +23,8 @@
 
   describe "when the app call results in a raised throttle" do
     before do
-      @app.expects(:call).with(@env).raises(Prop::RateLimited.new(:handle => "foo", :threshold => 10, :interval => 60, :cache_key => "wibble", :description => "Boom!"))
+      @app.expects(:call).with(@env).raises(Prop::RateLimited.new(:handle => "foo", :threshold => 10, :interval => 60, :cache_key => "wibble", :description => "Boom!", :strategy => Prop::IntervalStrategy
+                                            ))
```

Comment:
This looks weird with the `))` on the second line, just put it on the first line or split into multiple lines